FoodOrder.TestApi/Services/RestaurantTestServices.cs

```diff
@@ -0,0 +1,116 @@
+﻿using AutoMapper;
+using FluentAssertions;
+using FoodOrderApi.Controllers;
+using FoodOrderApi.DataProvider;
+using FoodOrderApi.Mappings;
+using FoodOrderApi.Repository;
+using FoodOrderApi.TestApi.Mockdata;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace FoodOrderApi.TestApi.Services
+{
+    public class TestRestaurantServices : IDisposable
+    {
+        protected readonly FakeFoodApiDbContext _context;
```

Comment:
Why protected?

Reply:
It is generate with the help of Intellisense while creating the constructor.